examples/todo/js/mutations/AddTodoMutation.js

```diff
@@ -33,6 +29,8 @@ export default class AddTodoMutation extends Relay.Mutation {
       edgeName: 'todoEdge',
       rangeBehaviors: {
         '': 'append',
+        'complete()': 'append',
```

Comment:
@steveluscher @josephsavona

I don't understand this part and whether it's correct at all.

The `orderby:newest` example on https://facebook.github.io/relay/docs/guides-mutations.html#range-add-example seems to be actively misleading w/r/t the form of calls... that's easy enough to figure out in a debugger.

However, I can't square what I'm seeing with https://github.com/facebook/relay/issues/293#issuecomment-140782479. If I take off the explicit `append` behaviors, I don't see anything happen at all to the `todos` connection. However, even in the `complete(true)` case, I don't see the `todos` connection get re-fetched in its entirety.

If I add the naked `todos` field to the fat query, I instead see the entire `todos` connection get sent down from the server no matter what I set up in my `rangeBehaviors`.

What am I not understanding here? In general, is there a way to conditionally re-fetch the entire connection with a `RANGE_ADD` mutation?